templates/data/opensearch/index.html

```diff
@@ -0,0 +1,336 @@
+            url="https://assets.ubuntu.com/v1/c0bf7b88-opensearch-consulting-services-illustration.svg",
+            alt="", 
+            width="240", 
+            height="137",
+            hi_def=True, 
+            loading="lazy" 
+          ) | safe 
+        }}
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip u-no-padding--top">
+  <div class="row">
+    <hr class="u-no-margin--bottom" />
+    <div class="col-6">
+      <div class="p-strip is-shallow u-no-padding--top">
+        <h2>Stand-alone <br class="u-hide--small" />OpenSearch<sup>&reg;</sup> Canonical services</h2>
+      </div>
+    </div>
+    <div class="col-9 col-start-large-4">
+      <hr class="u-no-margin--bottom" />
+      <div class="row">
+        <div class="col-3 col-medium-3">
+          <h3 class="p-heading--5">Security patching</h3>
+        </div>
+        <div class="col-6 col-medium-3">
+          <p>
+            Security patching for your OpenSearch application for critical and high Common Vulnerabilities and Exposures (CVEs) for your OpenSearch software.
+          </p>
+        </div>
+      </div>
+      <hr class="u-no-margin--bottom" />
+      <div class="row">
+        <div class="col-3 col-medium-3">
+          <h3 class="p-heading--5">Support</h3>
+        </div>
+        <div class="col-6 col-medium-3">
+          <p>
+            24/7 support and professional services ensure your initiatives are always on track. This includes bug-fixes and operational support for your OpenSearch cluster.
+          </p>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip u-no-padding--top">
+  <div class="row">
+    <hr class="u-no-margin--bottom" />
+    <div class="col-6 col-medium-3">
+      <h2>Fully automated <br class="u-hide--small" />application operations with Charmed OpenSearch</h2>
+    </div>
+    <div class="col-6 col-medium-3">
+      <p>
+        OpenSearch is a fantastic open source search and analytics suite. However, operating and supporting a production OpenSearch solution can be complex and challenging.
+      </p>
+      <p>
+        Canonical and the community are developing an open source operator called  Charmed OpenSearch, making it easier to operate OpenSearch on physical,  Virtual Machines (VM) and other wide range of cloud and cloud-like environments in an automated way. 
+      </p>
+      <p>
+        The Charmed OpenSearch operator is available for the open source community to use. Learn more about the ‘edge’ version of the operator in the  <a href="https://charmhub.io/opensearch">Charmed OpenSearch Technical documentation</a>.
+      </p>
+      <p>
+        <a href="/data/opensearch#get-in-touch" class="js-invoke-modal">
+          Notify me when the operator enters General Availability&nbsp;&rsaquo;
+        </a>
+      </p>
+      <div class="p-strip--light u-no-padding--top u-no-padding--bottom u-align--center u-vertically-center u-aspect-ratio--3-2">
```

Comment:
This image isn't hidden on small screens whereas the others are, is that the intended design?